addons/mrp/models/stock_move.py

```diff
@@ -436,6 +438,13 @@ def write(self, vals):
             self.filtered(lambda m: m.raw_material_production_id.state == 'confirmed')._run_procurement(old_demand)
         return res
 
+    def _get_moves_to_assign(self):
```

Comment:
This part seems more over engineered than really needed. What do you try to achieve?